src/spikeinterface/core/core_tools.py

```diff
@@ -934,3 +941,40 @@ def convert_bytes_to_str(byte_value: int) -> str:
         byte_value /= 1024
         i += 1
     return f"{byte_value:.2f} {suffixes[i]}"
+
+
+def spike_vector_to_dict(spike_vector: np.ndarray) -> dict:
+    assert HAVE_NUMBA
```

Comment:
```suggestion
    assert HAVE_NUMBA, ('numba must be installed to use spike_vector_to_dict with "
    "from_spike_vector=True" set "from_spike_vector=False" to use without numba')
```

I think having a more helpful assert for users that try to precompute will be necessary since installing core will not automatically install numba. This is just a first draft assert statement to help us track where the error is coming from.

Reply:
I don't like the message here (because maybe this function will be used by another method elsewhere?)

I'll make a commit to have a better error message :)

Reply:
Cool. and thanks! 

Yeah it is annoying/poor form to reference another function here. It would be better to have one assert in the original function, but we can't survey for numba there and a separate assert here to also make sure numba is useable.